src/Conf.ml

```diff
@@ -1737,14 +1739,15 @@ let kind_of fname =
   | _ -> !kind
 
 let xdg_config =
-  match Caml.Sys.getenv_opt "XDG_CONFIG_HOME" with
-  | Some xdg_config_home ->
-      let filename =
-        Fpath.(
-          append (v xdg_config_home) (v "ocamlformat" / dot_ocamlformat))
-      in
-      if Fpath.exists filename then Some filename else None
-  | None -> None
+  let xdg_config_home =
+    match Caml.Sys.getenv_opt "XDG_CONFIG_HOME" with
+    | None | Some "" ->
+        let home = Option.value ~default:"" (Caml.Sys.getenv_opt "HOME") in
+        Fpath.(v home / ".config")
+    | Some xdg_config_home -> Fpath.v xdg_config_home
+  in
+  let filename = Fpath.(xdg_config_home / "ocamlformat") in
```

Comment:
Shouldn't this be `xdg_config_home / "ocamlformat" / dot_ocamlformat` ?

Reply:
It was suggested in the issue to write the conf in an `ocamlformat` file instead of an hidden `.ocamlformat` file, I agree with that, but if I'm alone on this one we can go back to an `.ocamlformat` file.

Reply:
The rest of the docs refer to `.ocamlformat`, we shouldn't change the name depending on the directory.

Reply:
The doc has been updated according to this change, but do you want to keep the current architecture: `xdg_config_home / "ocamlformat" / dot_ocamlformat`?

Reply:
Sorry I didn't see your suggestion. I agree `xdg_config_home / "ocamlformat"` makes sense.
@jberdine I don't think we should put files starting with a dot in ".config" or sub-directories, I think it is a common pattern to remove the "." inside ".config"

Reply:
I don't agree, but won't make an issue of it.